test/test.cc

```diff
@@ -2855,13 +2869,13 @@ TEST_F(ServerTest, GetStreamedEndless) {
 TEST_F(ServerTest, ClientStop) {
   std::vector<std::thread> threads;
   for (auto i = 0; i < 3; i++) {
-    threads.emplace_back(thread([&]() {
+    threads.emplace_back([&]() {
       auto res = cli_.Get("/streamed-cancel",
                           [&](const char *, uint64_t) { return true; });
       ASSERT_TRUE(!res);
       EXPECT_TRUE(res.error() == Error::Canceled ||
                   res.error() == Error::Read || res.error() == Error::Write);
-    }));
+    });
   }
 
   std::this_thread::sleep_for(std::chrono::seconds(2));
```

Comment:
@yhirose For some reason, `ServerTest.ClientStop` failed and ran indefinitely (please see [results](https://github.com/yhirose/cpp-httplib/actions/runs/4539121445/jobs/7998678443)). I'm not sure what could be causing this problem. My change to `ServerTest.ClientStop` was trivial.